homeassistant/components/tuya/sensor.py

```diff
@@ -962,6 +989,7 @@ def native_value(self) -> StateType:
         """Return the value reported by the sensor."""
         # Only continue if data type is known
         if self._type not in (
+            DPType.BITMAP,
```

Comment:
How would it handle the bitmap?

Reply:
The value is just exposed via the sensor, you can use template binary sensors to pull out specific "bits" - e.g. "all ok", "not enough water", etc. In general, for sous vide cooker it doesn't actually behave as a bitmap. 1 means "all ok", 3 means "not enough water".

Reply:
> The value is just exposed via the sensor, you can use template binary sensors to pull out specific

Users should not have to learn about bitwise operators and templates to access these values.

Reply:
Ok.
I am more than happy to simply exclude this sensor but I don't have the time to implement bitmap support in this integration properly unfortunately.

Reply:
>  I don't have the time to implement bitmap support in this integration properly unfortunately

In that case, I suggest not adding it at all.

Reply:
> > I don't have the time to implement bitmap support in this integration properly unfortunately
> 
> In that case, I suggest not adding it at all.

Removed